src/averageAges.js

```diff
@@ -15,6 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const peopleMen = people.filter((person) => person.sex === 'm'
+    && (Math.ceil(person.died / 100) === century || !century));
+  const ageMen = peopleMen.map((personMen) => personMen.died - personMen.born);
+  const averageAgeMen = (ageMen.reduce((sumAge, current) =>
+    sumAge + current)) / ageMen.length;
+
+  return averageAgeMen;
```

Comment:
You repeat this logic several times. So create a separate method for calculating an average age